pubspec.yaml

```diff
@@ -27,6 +27,7 @@ dependencies:
     git:
       url: git@github.com:lenra-io/lenra_components.git
       ref: v1.0.0-beta.52
+  url_launcher: ^6.1.11
```

Comment:
Why do we need this dependency ?

Reply:
@taorepoara This is necessary to properly open the lenra.io website on every platform when clicking on the little information button.
![image](https://github.com/lenra-io/client-common/assets/36544012/a9cc6afd-16b3-45d9-ad6e-28a86e0f7732)

Reply:
I forgot this button ^^